Sources/SourceKitLSPAPI/BuildDescription.swift

```diff
@@ -160,12 +158,45 @@ public struct BuildDescription {
 
     /// The inputs of the build plan so we don't need to re-compute them  on every call to
     /// `fileAffectsSwiftOrClangBuildSettings`.
-    private let inputs: [BuildPlan.Input]
+    private let inputs: [Build.BuildPlan.Input]
+
+    public static func load(
+      location: Workspace.Location,
+      destinationBuildParameters: BuildParameters,
+      toolsBuildParameters: BuildParameters,
+      packageGraphLoader: @escaping () async throws -> ModulesGraph,
+      pluginConfiguration: PluginConfiguration,
+      disableSandbox: Bool,
+      fileSystem: any FileSystem,
+      observabilityScope: ObservabilityScope
+    ) async throws -> (description: BuildDescription, errors: String) {
+        let bufferedOutput = BufferedOutputByteStream()
+        let threadSafeOutput = ThreadSafeOutputByteStream(bufferedOutput)
+
+        // This is quite an abuse of `BuildOperation`, building plugins should really be refactored out of it.
```

Comment:
question: Will a GH issue be created to fix this?

Reply:
Well, the issue will likely be for creating a BSP server instead (in which case what SwiftPM is doing internally doesn't *really* matter), but yes.